opm/material/components/co2tables.inc

```diff
@@ -40864,13 +40867,13 @@ struct CO2Tables {
    static const double brineSalinity;
 };
 
-TabulatedFunction CO2Tables::tabulatedEnthalpy;
-TabulatedFunction CO2Tables::tabulatedDensity;
-const double CO2Tables::brineSalinity = 1.000000000000000e-01;
+inline TabulatedFunction CO2Tables::tabulatedEnthalpy;
+inline TabulatedFunction CO2Tables::tabulatedDensity;
+inline const double CO2Tables::brineSalinity = 1.000000000000000e-01;
```

Comment:
Is it possible to make this imput in a simpe way?

Reply:
Currently this is a auto-generated file based on an old Span-Wagner code. We could look into if we could change the output to something more readable.